src/elements/earn/portfolio/v3/initWithdraw/step4/V3WithdrawStep4.tsx

```diff
@@ -1,13 +1,49 @@
 import { Button, ButtonVariant } from 'components/button/Button';
 import { ReactComponent as IconCheck } from 'assets/icons/circlecheck.svg';
+import { ReactComponent as IconBell } from 'assets/icons/bell.svg';
+import { ReactComponent as IconCalendar } from 'assets/icons/calendar.svg';
 import { memo } from 'react';
+import {
+  CalendarOptions,
+  GoogleCalendar,
+  ICalendar,
+  OutlookCalendar,
+} from 'datebook';
+import { PopoverV3 } from 'components/popover/PopoverV3';
+import { openNewTab } from 'utils/pureFunctions';
+import dayjs from 'dayjs';
+
+const generateCalendarEvent = (
+  type: 'ical' | 'google' | 'outlook',
+  days: number
+) => {
+  const start = dayjs().add(days, 'day').toDate();
+  const end = dayjs(start).add(15, 'minute').toDate();
+
+  const event: CalendarOptions = {
+    title: 'Bancor Withdrawal',
+    description: 'Bancor Withdrawal cooldown ended.',
```

Comment:
Inconsistent casing in the description. Should probably be withdrawal with lowercase w

Reply:
fixed